### PR TITLE
vmctl/vm: added datapoints collection bar

### PR DESCRIPTION
**Pull request by dmitryk-dk**

### CODE REVIEW

File: app/vmctl/influx.go [hunk 1]
```diff
@@ -44,7 +44,10 @@ func (ip *influxProcessor) run(silent, verbose bool) error {
 		return nil
 	}
 
-	bar := pb.StartNew(len(series))
+	bar := progressbar.AddWithTemplate(progressbar.ProgressTemplate("Processing series"), len(series))
+	if err := progressbar.Start(); err != nil {
+		log.Printf("error start process bars pool: %s", err)
```
Comment (f41gh7) [inline app/vmctl/influx.go:49]: I suggest to return error if bar cannot start, otherwise, bar may throw panics.

**Comment by qiu-peng20** [comment]
@dmitryk-dk Hi，bro
I tried to use your branch code, but still got stuck at 0% 
image

    PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND
   106 root      20   0 1504.2g   1.2g   1.1g S   0.0  15.8   0:05.82 vmctl

**Comment by dmitryk-dk** [comment]
Hi @qiu-peng20 ! Thank for your screenshot! I will try to reproduce your problem. What i see from your screenshot that you have 40 blocks and only one is trying to import, maybe problem with this. I will check

**Comment by dmitryk-dk** [comment]
@qiu-peng20 can you try to import your data into local victoriametrics, just to be sure that vmctl really blocks?

**Comment by dmitryk-dk** [comment]
@qiu-peng20 I have tried simulate your problem and it works well, please check the screen below.
Screenshot 2022-04-29 at 14 14 09

**Comment by hagen1778** [comment]
> I tried to use your branch code, but still got stuck at 0%

Hi @qiu-peng20! Thanks for your feedback. It says that VM is not the bottleneck here, since importer does not receive any samples to import. The `vmctl prometheus` import tool consists of two parts:
1. prometheus snapshot reader
2. vm importer

The reader (p1) reads the snapshot data from the disk and forwards found samples to the importer. Importer (p2) forms the batches and pushes them to VM. It is clear, that reader fails to deliver samples to the importer, which is weird since from your comments it is able to read snapshots metadata (the stats info before the import process). Is it possible for you to try to re-export Prometheus snapshot and try again? 

Please, let's continue conversation in the https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2469

**Comment by qiu-peng20** [comment]
Excuse me, why does the transmission efficiency gradually decrease over time?
start：
image

After a while：
image

**Pull request merged**